### DISCUSSION TITLE
Unable to find an inverter

### DISCUSSION THREAD

**Discussion by miralemR** [nathanmarlor/foxess_modbus#517] [Support]
Hello.
When I want to add a card on dashboard I get this message

Unable to find an inverter with the device ID or friendly name ''. Valid friendly names: 'Ngen H3'

Should I reinstall Foxess modbus integration an leave name window empty?

**Comment by canton7** [comment]
Are you referring to the charge period card? https://github.com/nathanmarlor/foxess_modbus_charge_period_card

**Comment by miralemR** [comment]
Yes.

V sre., 17. jan. 2024, 17:17 je oseba Antony Male ***@***.***>
napisala:

**Reply by canton7** [reply]
So, use  'Ngen H3' as the friendly name of the inverter to use when you configure the card, or select your inverter from the drop-down.

**Comment by miralemR** [comment]
Hello again. I am still having problems with the charge periods card. I
have been searching for problems that might be on my configuration
specifically but can't seem to find them. I have tried to select the
inverter from the drop-down but it doesn't show up. I also tried to insert
it in the code editor but the card just says loading and never loads (I
have waited a few days to be sure).
I have added screenshots of my charge periods card configuration if it is
of any help. "Ni ustreznih naprav" means "no suitable devices".
Thank you for your help.

V V sre., 17. jan. 2024 ob 18:27 je oseba Antony Male <
***@***.***> napisala:

**Reply by canton7** [reply]
The H3 does not support setting charge periods over modbus I'm afraid

**Comment by miralemR** [comment]
What about the Foxess- energy management integration? I have an ethernet
adapter as a modbus host and I connect to it with rs485. Foxess energy
management  says that it needs to be RTU -> TCP, not direct LAN. When I
insert the IP address it gives an error. What am I doing wrong / is there
any workaround or...?

V V ned., 10. mar. 2024 ob 18:23 je oseba Antony Male <
***@***.***> napisala:

**Reply by canton7** [reply]
For foxess_em questions, please raise a discussion on the foxess_em repo.
